docs/user/dashboard/edit-dashboards.asciidoc

```diff
@@ -0,0 +1,114 @@
+[[edit-dashboards]]
+== Edit dashboards
+
+Now that you have added panels to your dashboard, you can add filter panels to interact with the data, and Markdown panels to add context to the dashboard. 
+To make your dashboard look the way you want, use the editing options.
+
+[float]
+[[add-controls]]
+=== Add controls
+
+To filter the data on your dashboard in real-time, add a *Controls* panel.
+
+You can add two types of *Controls*:
+
+* Options list — Filters content based on one or more specified options. The dropdown menu is dynamically populated with the results of a terms aggregation. 
+For example, use the options list on the sample flight dashboard when you want to filter the data by origin city and destination city.
+
+* Range slider — Filters data within a specified range of numbers. The minimum and maximum values are dynamically populated with the results of a 
+min and max aggregation. For example, use the range slider when you want to filter the sample flight dashboard by a specific average ticket price.
+
+[role="screenshot"]
+image::images/dashboard-controls.png[]
+
+To configure *Controls* for your dashboard:
+
+. Click *Options*, then configure the following:
+
+* *Update Kibana filters on each change* &mdash; When selected, all interactive inputs create filters that refresh the dashboard. When unselected,
+ {kib} filters are created only when you click *Apply changes*.
+
+* *Use time filter* &mdash; When selected, the aggregations that generate the options list and time range are connected to the <<set-time-filter,time filter>>.
+
+* *Pin filters to global state* &mdash; When selected, all filters created by interacting with the inputs are automatically pinned.
+
+. Click *Update*.
+
+[float]
+[[add-markdown]]
+=== Add Markdown
+
+*Markdown* is a text entry field that accepts GitHub-flavored Markdown text. When you enter the text, the tool populates the results on the dashboard. 
+
+Use Markdown when you want to add context to the other panels on your dashboard, such as important information, instructions and images.
+
+For information about GitHub-flavored Markdown text, click *Help*.
```

Comment:
maybe "click Help in the Mardown window (or whatever it is called)."

Reply:
It's the only *Help* option on the UI.